src/Entity/AddressValidation/AVAddress.php

```diff
@@ -59,14 +67,24 @@ public function __construct(\SimpleXMLElement $xmlDoc)
         $this->addressClassification = isset($xmlDoc->AddressClassification) ? new AddressClassification($xmlDoc->AddressClassification) : null;
         $this->consigneeName = isset($xmlDoc->ConsigneeName) ? (string)$xmlDoc->ConsigneeName : '';
         $this->buildingName = isset($xmlDoc->BuildingName) ? (string)$xmlDoc->BuildingName : '';
-        $this->addressLine = isset($xmlDoc->AddressLine) ? (string)$xmlDoc->AddressLine : '';
-        $this->region = isset($xmlDoc->Region) ? (string)$xmlDoc->Regions : '';
+        if (isset($xmlDoc->AddressLine)) {
```

Comment:
It's definitely not the sexiest 8 lines of code I've ever written. I appreciate your solution--quite a bit more elegant. I'll swap it out.